### PR TITLE
[OCPBUGS7600] Align IBM Z install documentation 

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.9 +

Issue: 
https://issues.redhat.com/browse/OCPBUGS-7600

Link to docs preview:
- [z/VM restricted - Preparing the user-provisioned infrastructure](https://56025--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-restricted-networks-ibm-z.html#installation-infrastructure-user-infra_installing-restricted-networks-ibm-z)
- [RHEL KVM restricted - Networking requirements for user-provisioned infrastructure](https://56025--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.html#installation-network-connectivity-user-infra_installing-restricted-networks-ibm-z-kvm)
-  [z/VM restricted - Networking requirements for user-provisioned infrastructure](https://56025--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-restricted-networks-ibm-z.html#installation-network-user-infra_installing-restricted-networks-ibm-z)

QE review:
- [x] QE has approved this change.

Additional information: From a thorough comparison we did for the different IBM Z install docs we noticed some bugs that need correction. Typos, Style Guide, and incorrect conditionals in some modules 
comparison doc: https://docs.google.com/document/d/13US8rh7dK4b2WLVRmChhA56jIcyXshO5BzmttMRZa34/edit

### CODE REVIEW

**Comment by holgwolf** [comment]
LGTM

**Comment by clapolt** [comment]
LGTM

File: modules/preferred-ibm-z-system-requirements.adoc [hunk 1]
```diff
@@ -5,31 +5,31 @@
...
 [id="preferred-ibm-z-system-requirements_{context}"]
-= Preferred IBM Z system environment
+= Preferred {ibmzProductName} system environment
 
 [discrete]
 == Hardware requirements
 
-* 3 LPARS that each have the equivalent of 6 IFLs, which are SMT2 enabled, for each cluster.
-* Two network connections to connect to both connect to the `LoadBalancer` service and to serve data for traffic outside the cluster.
+* Three LPARS that each have the equivalent of six IFLs, which are SMT2 enabled, for each cluster.
```
Comment (JoeAldinger) [inline modules/preferred-ibm-z-system-requirements.adoc:13]: good catch with these numbers.

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Pull request merged**